### PR TITLE
github-pages: add caveat about --force push

### PR DESCRIPTION
**Pull request by timwis**
Ref:
- #1064 
- travis-ci/travis-ci#7562
/cc @webknjaz

### CODE REVIEW

File: user/deployment/pages.md [hunk 1]
```diff
@@ -11,6 +11,13 @@ You will need to provide a [personal access
 token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/)
 and set the deployment provider details in `.travis.yml`.
 
+> Note that Travis CI will [`--force` push](https://github.com/travis-ci/dpl/blob/bdc795f335e23c3f470fbda89aa4a173d79dd424/lib/dpl/provider/pages.rb#L52-L62)
+> to the target branch (ie. `gh-pages`), replacing any files there and overwriting
```
Comment (webknjaz) [inline user/deployment/pages.md]: > ie.

i.e.

File: user/deployment/pages.md [hunk 2]
```diff
@@ -11,6 +11,13 @@ You will need to provide a [personal access
...
+> the history of that branch. This should be fine if you only use the target branch
+> for deployments. It won't overwrite the source branch (ie. `master`). Can you
```
Comment (webknjaz) [inline user/deployment/pages.md]: > It won't overwrite the source branch (ie. `master`).

It won't overwrite the source branch (i.e. `master`), unless you **specify it as a target branch**.

Comment (danielo515) [inline user/deployment/pages.md]: I would also add:
> unless you specify it as a target branch too

**Comment by plaindocs** [comment]
Oops, I missed this.  Thanks for the work. 

Closing in favor of https://github.com/travis-ci/docs-travis-ci-com/pull/1184 and I'll update that PR.

**Pull request closed without merging**